app/views/home/index.html.erb

```diff
@@ -1 +1,31 @@
-<h1 class="text-blue-500">Hello, world!<h1>
\ No newline at end of file
+<div class="flex flex-col items-center px-10 md:px-32 lg:px-60 py-10 lg:py-14">
+  <h1 class="mb-7 text-primary">¡Hola!</h1>
+  <h5 class="mb-7">Te damos la bienvenida a</h5>
+
+  <form class="pb-12 w-full">
```

Comment:
I suggest using the Rails helper `form_tag` it should make easier to add the backend functionality later.

Reply:
Yeah, I was unsure to use it or not because I don't quite understand how we will send the address to a Geolocation API and then use it in the app without saving it in a DB.
I'll make the change then!